.github/workflows/main.yml

```diff
@@ -21,9 +21,9 @@ jobs:
             activate_cmd: "activate"
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Setup conda
-      uses: s-weigand/setup-conda@master
```

Comment:
The action was not locating `s-weigand/setup-conda@master`, used the value in the repo's readme. also updated checkout to the version present in the example there.